### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by HanSolo**
Added a github action to run mvn verify on each push on Ubuntu 18.04 using JDK 8

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [8]
```
Comment (hannesa2) [inline .github/workflows/ci.yml:12]: What's the point to have a matrix for `os` and `java` having only one single value ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      fail-fast: false
+      max-parallel: 4
```
Comment (hannesa2) [inline .github/workflows/ci.yml:14]: Without a valid matrix this is pointless too. It seems it's a copy&paste leftover

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
```
Comment (hannesa2) [inline .github/workflows/ci.yml:19]: As far as I know, `ubuntu-18.04` has already java 8 installed

**Review by hannesa2 — APPROVED** [review]
Anyway, I like to see a CI here.
Maybe you find some time to simplify and reduce to necessary commands ?

**Comment by joelittlejohn** [comment]
We actually use Travis for CI already. It's there any benefit to moving to GitHub actions?

**Comment by hannesa2** [comment]
> We actually use Travis for CI already. It's there any benefit to moving to GitHub actions?

It's faster, it can keep some artifacts, you have a lot of additional actions, you can split tasks and you can run Espresso tests with ease. 

Maybe more advantages are there too

**Comment by hannesa2** [comment]
Give it a try, the one doesn't exclude the other

File: .github/workflows/ci.yml (hunk 1 above)
Comment (HanSolo) [reply .github/workflows/ci.yml:12]: Just left the matrix in, in case you would like to add other operating systems or jdk‘s to test against.

**Pull request merged**